server/jetstream.go

```diff
@@ -48,22 +48,22 @@ type JetStreamConfig struct {
 	Domain    string `json:"domain,omitempty"`
 }
 
+// Statistics about JetStream for this server.
 type JetStreamStats struct {
-	Memory             uint64            `json:"memory"`
-	Store              uint64            `json:"storage"`
-	ReservedMemoryUsed uint64            `json:"reserved_memory_used,omitempty"`
-	ReserveStoreUsed   uint64            `json:"reserved_storage_used,omitempty"`
-	Accounts           int               `json:"accounts,omitempty"`
-	API                JetStreamAPIStats `json:"api"`
-	ReservedMemory     uint64            `json:"reserved_memory,omitempty"`
-	ReservedStore      uint64            `json:"reserved_storage,omitempty"`
+	Memory         uint64            `json:"memory"`
+	Store          uint64            `json:"storage"`
+	ReservedMemory uint64            `json:"reserved_memory"`
+	ReservedStore  uint64            `json:"reserved_storage"`
```

Comment:
Can these changes really be done at this point given semver?

Reply:
We discussed yesterday, not totally sure yet, but the reserved portions were incorrect and needed to be fixed. The one issue identified would be the nats cli most likely.

Reply:
Is it deliberate that you removed the `,omitempty`? (for both ReservedMemory and ReservedStore)

Reply:
Could you comment on this question: "Is it deliberate that you removed the ,omitempty? (for both ReservedMemory and ReservedStore)"

Reply:
Yes it was deliberate based on feedback and my usage from varz/jsz and statusz stuff.

Reply:
Ok